packages/terra-application-navigation/tests/wdio/application-navigation-spec.js

```diff
@@ -0,0 +1,150 @@
+Terra.describeViewports('ApplicationNavigation - Large', ['large'], () => {
+  describe('Extensions roll up properly', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-application-navigation/application-navigation/extensions');
```

Comment:
We should update this spec to use `it` blocks instead of before hooks

Reply:
Updated.